circle.yml

```diff
@@ -9,15 +9,15 @@ machine:
 
 dependencies:
   override:
-    - ./gradlew --parallel resolveConfigurations
+    - ./gradlew --parallel --no-daemon resolveConfigurations
```

Comment:
add comment: `# Cannot use daemon mode since standard build and verify-ibmjvm.sh build use different plugins and classpaths, see build.gradle`